compiler/rustc_infer/src/infer/outlives/for_liveness.rs

```diff
@@ -57,7 +57,7 @@ where
             //
             // We search through the item bounds and where clauses for
             // either `'static` or a unique outlives region, and if one is
-            // found, we just need to prove that that region is still live.
```

Comment:
This is correct, please don't change.

I won't go over the rest of the PR. When you submit things like this, please ensure the things you are fixing are indeed typos.